CLI.md

```diff
@@ -214,6 +216,20 @@ Open (i.e. activate) a new multi-owner chain deriving the UID from an existing o
 
 
 
+## `linera change-application-permissions`
+
+Changes the application permissions configuration
+
+**Usage:** `linera change-application-permissions [OPTIONS] --chain-id <CHAIN_ID>`
+
+###### **Options:**
+
+* `--chain-id <CHAIN_ID>` — The id of the chain to which the new permissions will be applied
```

Comment:
Nit: should we use an upper case "ID"? (Not sure how it's written in other sections though :sweat_smile:)

```suggestion
* `--chain-id <CHAIN_ID>` — The ID of the chain to which the new permissions will be applied
```

Reply:
That's how I'd normally spell it; I just saw another "chain id" in a CLI argument description and used it. But now I'm also seeing "ID"s there… probably my bad.

Reply:
Let's discuss with the team and fix the inconsistency in a separate PR.